methods.py

```diff
@@ -808,7 +808,7 @@ def get_compiler_version(env):
     if env.msvc and not using_clang(env):
         try:
             args = [env["VSWHERE"], "-latest", "-products", "*", "-requires", "Microsoft.Component.MSBuild"]
```

Comment:
```suggestion
            args = [env["VSWHERE"], "-latest", "-products", "*", "-requires", "Microsoft.Component.MSBuild", "-utf8"]
```
Turns out you can just do this :)
https://github.com/microsoft/vswhere/wiki/Encoding

Reply:
Oh. Well, that makes things easier